docs/topics/native/native-memory-manager.md

```diff
@@ -4,25 +4,29 @@ Kotlin/Native uses a modern memory manager that is similar to the JVM, Go, and o
 the following features:
 
 * Objects are stored in a shared heap and can be accessed from any thread.
-* Tracing garbage collection (GC) is performed periodically to collect objects that are not reachable from the "roots",
+* Tracing garbage collection is performed periodically to collect objects that are not reachable from the "roots",
   like local and global variables.
 
 ## Garbage collector
 
-Kotlin/Native's GC algorithm is constantly evolving. Currently, it functions as a stop-the-world mark and concurrent sweep
-collector that does not separate the heap into generations.
+Kotlin/Native's garbage collector (GC) algorithm is constantly evolving. Currently, it functions as a stop-the-world mark
+and concurrent sweep collector that does not separate the heap into generations.
 
-The GC uses a full parallel mark that combines paused mutators, the GC thread, and optional marker threads to process
-the mark queue. By default, paused mutators and at least one GC thread participate in the marking process.
-You can disable the full parallel mark with the `-Xbinary=gcMarkSingleThreaded=true` compilation option.
-However, this may increase the pause time of the garbage collector.
+The GC is executed on a separate thread and started based on the timer
+and memory pressure heuristics. Alternatively, it can be [called manually](#enable-garbage-collection-manually).
+
+To process the mark queue, the GC uses a full parallel mark that combines application threads, the GC thread,
+and optional marker threads. Application threads and at least one GC thread participate in the marking process.
+By default, application threads must be paused when the GC is marking objects in the heap.
+
+> You can disable the full parallel mark with the `-Xbinary=gcMarkSingleThreaded=true` compiler option.
+> However, this may increase the garbage collector's pause time.
```

Comment:
And then here: "You can disable the parallelization of the mark phase with the `-Xbinary=gcMarkSingleThreaded=true` compiler option. However, this may increase the garbage collector's pause time on large heaps."